arches/app/media/js/views/components/cards/photo-gallery-card.js

```diff
@@ -59,6 +59,16 @@ define([
             return mfs;
         });
 
+        this.acceptedFiles = ko.computed(function(){
+            var aft = "Missing filetypes";
```

Comment:
It looks like we will never use this string. Given that, maybe this could be something like:
```
this.acceptedFiles = ko.computed(function(){
    return self.card.widgets().find(widget=>widget.node_id() === self.fileListNodeId)?.config.acceptedFiles() || '--';
});
 ```

Reply:
I've changed the line logic within the computed function as suggested.  I've also changed -- to All Formats Accepted as I think the double dashes might give be impression that no formats are accepted.